pkg/ccl/multitenantccl/tenantcostclient/tenant_side.go

```diff
@@ -34,6 +34,16 @@ import (
 	"github.com/cockroachdb/errors/errorspb"
 )
 
+// InitialRequestSetting is exported for testing purposes.
+var InitialRequestSetting = settings.RegisterFloatSetting(
+	settings.SystemVisible,
+	"tenant_initial_request",
+	"number of request units to get from server on first request (requires restart)",
+	bufferRUs/5,
+	settings.WithName("tenant_cost_control.initial_request"),
```

Comment:
Note for the future: there is no need to add WithName for a _new_ cluster setting.  The WithName option allows renaming an existing setting.  Since this has already merged to master, I don't think we should change this as part of the backport.

Reply:
I can fix that up in `master`, but won't worry about the backports.